admin/modules/config/thread_prefixes.php

```diff
@@ -510,14 +510,65 @@ function checkAction(id)
 
 	$table = new Table;
 	$table->construct_header($lang->prefix);
+	$table->construct_header($lang->forums);
 	$table->construct_header($lang->controls, array('class' => 'align_center', 'colspan' => 2));
 
 	$prefixes = build_prefixes();
+
+	foreach($prefixes as &$prefix)
+	{
+		$prefix['forum_fids'] = explode(',', $prefix['forums']);
+	}
+	unset($prefix);
+
+	$forums = cache_forums();
```

Comment:
The [`get_forum()`](https://github.com/mybb/mybb/blob/mybb_1822/inc/functions.php#L6267) that wraps details of fetching forums data, IMO, is better than retrieving all forums data. Anyway, using `cache_forums()` is valid.